.github/workflows/build.yml

```diff
@@ -18,19 +18,19 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - name: Setup Go
-        uses: actions/setup-go@v1
+        uses: actions/setup-go@v5
         with:
-          go-version: 1.19
+          go-version: 1.21
```

Comment:
maybe is better to specified patch version 1.21.6 to include latest security fixes